modules/main-rules/pages/the-start-phase.adoc

```diff
@@ -0,0 +1,26 @@
+= The Start phase
+
+The Start phase comprises preparatory steps which aren't always necessary -- the need for them is determined by what rules you use in your game, and what special orders you want to use for the new turn.
+
+== 1. Roll for Reserves
+
+If you are playing a scenario that uses the Reserves rules, then apply those rules now.
+
+== 2. Declare flyer missions and special orders
+
+There are some things that you must declare now if you want to use them:
+
+* Flyer missions.
+* Some types of special orders, such as Flak, Preparatory Bombardment and Dodge.
+
+Start with the player whose army has the lower Strategy Rating, and then alternate to declare until you both run out of things to declare.
+Take Leadership tests in the normal way, where the rules require it for you to successfully set the orders you declare.
+
+== 3. Prepare Initiative counters
```

Comment:
```suggestion
== 3. Prepare the Initiative counters
```

OK, I think this should give you a suggestion that can be accepted. I've added 'the' to match the text in the main turn sequence page.

Reply:
Yay, it's showing me a commit button, so you should be seeing it too.

Reply:
Yes, I see it, yay! And as long as I go to the **Files** tab first, I can **Add suggestion to batch** so that we get one commit rather than an excess of little ones — that's good.